src/server/DBMetadata/metadataProcessor.js

```diff
@@ -0,0 +1,77 @@
+const ProcessedField = require("./classes/processedField");
```

Comment:
Did this file change at all or did you just move it?

Reply:
@laueian I just moved/renamed it